html/cross-origin-embedder-policy/none-sw-from-none.https.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script>
+const SCOPE = new URL(location.href).pathname;
+const SCRIPT =
+  'resources/sw.js?' +
+  `pipe=header(service-worker-allowed,${SCOPE})`;
+
+function remote(path) {
+  const REMOTE_ORIGIN = get_host_info().HTTPS_REMOTE_ORIGIN;
+  return new URL(path, REMOTE_ORIGIN + '/html/cross-origin-embedder-policy/');
+}
+
+promise_test(async (t) => {
+  const reg = await service_worker_unregister_and_register(t, SCRIPT, SCOPE);
+  const sw =
+    reg.installing ? reg.installing : reg.waiting ? reg.waiting : reg.active;
+  await wait_for_state(t, sw, 'activated');
+}, 'setting up');
+
+promise_test(async (t) => {
+  await fetch('resources/nothing-same-origin-corp.txt', {mode: 'no-cors'});
+}, 'making a same-origin request for CORP: same-origin');
+
+promise_test(async (t) => {
+  await fetch('resources/nothing.txt', {mode: 'no-cors'});
```

Comment:
Any reason not to use `/common/blank.html`?

Reply:
I just forgot about it. Fixed.